exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -177,17 +201,23 @@ impl JavaServiceRuntime {
 
     /// Adds required EJB-related arguments to JVM configuration
     fn add_required_arguments(
-        args_builder: InitArgsBuilder,
+        mut args_builder: InitArgsBuilder,
         runtime_config: &RuntimeConfig,
+        internal_config: InternalConfig,
     ) -> InitArgsBuilder {
-        args_builder
-            .option(&format!(
+        // We do not use system library path in tests, because an absolute path to the native
+        // library will be provided at compile time using RPATH.
+        if internal_config.system_lib_path.is_some() {
+            args_builder = args_builder.option(&format!(
```

Comment:
What is the point of this assignment (and mutable args_builder)? Does this assignment have any
         effect?

Reply:
The Builder pattern in rust consumes original instance which fits great for a chain of calls but brings such tricks if we need to update a mutable argument.